api/models/user_permission.py

```diff
@@ -0,0 +1,35 @@
+from api.db import db
+
+
+class UserPermission(db.Model):
+    """
+    Manage Permission of Users
+    """
+    __tablename__ = 'user_permissions'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String, unique=True)
+    description = db.Column(db.String)
+    verified_user = db.Column(db.Boolean)
+    anonymous_user = db.Column(db.Boolean)
```

Comment:
Are we allowing the anonymous user? As it will only increase the load and they can simply register or use OAuth for registration